Framework/jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/SeleniumConfig.java

```diff
@@ -421,6 +423,22 @@ private static DesiredCapabilities getRemoteCapabilities() {
     return capabilities;
   }
 
+  /**
+   * Get the remote capabilities as a HashMap
+   * @return HashMap of remote capabilities
+   */
+  public static HashMap<String, String> getRemoteCapabilitiesAsStrings() {
+    return Config.getSection(SELENIUM_SECTION.RemoteSeleniumCapsMaqs);
```

Comment:
> Change this instance-reference to a static reference. - SonarCloud

Try introducing a private class field for the enum value and mark it static final.